src/components/todoList/View.js

```diff
@@ -54,8 +55,10 @@ class View extends React.Component {
                     </button>
                 </header>
 
-                <div><TodoAndCompleteItem todos={this.props.todos}/></div>
-                <div><ChooseFilter /></div>
+                <div className="todoAndDateSelect"><TodoAndCompleteItem /> 
+              
+                </div>
+                <div><ChooseFilter />  {store.getState().showSideBar? <RightSideBar/>:null} </div>
```

Comment:
this is not using redux store correctly, you have to use react-redux to connect to component. Otherwise, you have to implement your own subscribe.